docs/_features/skolemization.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: default
+title: Skolemization
+nav_order: 1
+---
+
+# Skolemization
+
+## What is Skolemization
+
+In the context of Linked Data, Skolemization is a process used to handle blank nodes or anonymous nodes in RDF 
+(Resource Description Framework) graphs. 
+These nodes, which lack unique identifiers, are frequently employed in RDF/S knowledge bases to represent complex 
+attributes or resources with known properties but unknown identities.
+
+Skolemization in Linked Data involves the transformation of these blank nodes into Skolem Uniform Resource Identifiers (URIs).
+The process enhances the clarity makes it easier to reference these nodes in future datasets.
+
+This process is particularly useful when dealing with substantial volumes of unstructured data distributed across 
+diverse sources. 
+By improving the accuracy and relevance of RDF summaries in relation to original datasets, 
+Skolemization enhances the efficiency and effectiveness of subsequent queries against these summaries.
+
+In summary, Skolemization in Linked Data provides a way to handle the complexity introduced by blank nodes in RDF graphs, 
+thereby enhancing the clarity, interoperability, and usability of the data.
+
+### Example
+
+Suppose we have the following RDF triples with a blank node represented as `_:`:
+
+```
+_:bnode <http://purl.org/dc/terms/title> "The Lord of the Rings" .
+_:bnode <http://purl.org/dc/terms/creator> "J.R.R. Tolkien" .
+```
+
+In this example, `_:` is a blank node that represents a resource (a book in this case) with known properties (title and creator) but an unknown identity.
+
+Through Skolemization, we can replace the blank node with a Skolem URI. 
+The Skolem URI is typically a URL that is unique to the blank node and is generated by the system handling the RDF data. 
+Here’s how it might look:
+
+```
+<http://example.org/.well-known/genid/123456> <http://purl.org/dc/terms/title> "The Lord of the Rings" .
+<http://example.org/.well-known/genid/123456> <http://purl.org/dc/terms/creator> "J.R.R. Tolkien" .
+```
+
+In this Skolemized version, the blank node has been replaced with the Skolem URI http://example.org/.well-known/genid/123456. 
+This URI is unique to the resource previously represented by the blank node, and can now be used to reference this 
+resource in other datasets. 
+This is a simple example, but it illustrates the basic process of Skolemization in Linked Data.
+
+## Skolemization in LDES Server
+
+To enable Skolemization, define the `<https://w3id.org/ldes#skolemizationDomain>` property when [setting up your Event Stream](../configuration/event-stream#configuring-a-new-event-stream) with for example with value `"http://example.org"`.
```

Comment:
It is very good this extensive explanation of skolemization is written here, however, when it comes to configuring the LDES, I would add this to the page provided in the URI, as it can get very confusing if all the config props are spread out. 
I would go with the approach of the eventSource and putting there a link to this page instead of the other way around. (Or in both ways could also work 😉)